src/dev.sh

```diff
@@ -304,13 +304,21 @@ function package ()
         tar_name="${agent_pkg_name}.tar.gz"
         echo "Creating $tar_name in ${LAYOUT_DIR}"
         tar -czf "${tar_name}" -C ${LAYOUT_DIR} .
+        echo "Calculate SHA256 checksum for $tar_name"
+        if [[ ("$PLATFORM" == "linux") ]]; then
+            sha256sum "${tar_name}" | cut -b 1-64 > ./Checksum_SHA256.txt
```

Comment:
- Would lowercase file name be more convenient for users? I really don't know, just posing the question for you to decide.
- Does this (or openssl) introduce additional dev dependencies that may not be part of the OS fresh install? Do we need to update any docs?
- I wonder how reliable the "cut" command is across different distros. Did you happen to see any forum posts confirming the accuracy of this across different distros?

I'm fine with the changes, double-check with Bryan if he has any concerns.